Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule.py

```diff
@@ -432,6 +437,7 @@ def feed_main(feed_name, params=None, prefix=''):
             args['feed_name'] = feed_name
             if feed_tags:
                 args['feedTags'] = feed_tags
+            args['tlp_color'] = params.get('tlp_color')
```

Comment:
Your code has `tlp_color` as an instance variable of the `Client` class, yet here you seem to be incorporating it into the `args` dictionary that is passed to command functions. Why?